Examples/DuvetExample/DuvetExample/Application/ViewControllers/StatusBarViewController.swift

```diff
@@ -0,0 +1,80 @@
+import Duvet
+import UIKit
+
+enum StatusBarStyle: CaseIterable {
+    static var allCases: [StatusBarStyle] {
+        var styles: [StatusBarStyle] = [.default, .light]
+        if #available(iOS 13.0, *) {
+            styles.append(.dark)
+        }
+        return styles
+    }
+
+    case `default`
+    case light
+
+    @available(iOS 13.0, *)
+    case dark
+
+    var label: String {
+        switch self {
+        case .default: return "Default"
+        case .light: return "Light"
+        case .dark: return "Dark"
+        }
+    }
+
+    var value: UIStatusBarStyle {
+        switch self {
+        case .default:
+            return .default
+        case .light:
+            return .lightContent
+        case .dark:
+            if #available(iOS 13.0, *) {
+                return .darkContent
+            } else {
+                return .default
+            }
+        }
+    }
+}
+
+/// A basic sheet view that showcases adjusting the status bar style from a SheetViewController's sheet items.
+class StatusBarViewController: BaseViewController, ProvidesSheetConfiguration {
+    static let sheetConfiguration = SheetConfiguration(
+        handleConfiguration: SheetHandleConfiguration(),
+        initialPosition: .fittingSize,
+        supportedPositions: [.fittingSize]
+    )
+
+    private var statusBarStyle: UIStatusBarStyle = .lightContent {
+        didSet {
+            setNeedsStatusBarAppearanceUpdate()
```

Comment:
Thanks for adding an example! Any way we could persist this during an app run? Then we'd also be able to see the transition when presenting a sheet too. Maybe a static var to maintain the value?